### PR TITLE
Change 'assert' handler to pass only upon boolean true is received

### PR DESCRIPTION
**Pull request by ruben-ayrapetyan**

### CODE REVIEW

File: tests/jerry-test-suite/12/12.11/12.11-002.js [hunk 1]
```diff
@@ -29,4 +29,4 @@ switch ("key") {
     break;
 }
 
-assert(matchesCount === 1 ? 1 : 0);
+assert(matchesCount === 1);
```
Comment (zherczeg) [inline tests/jerry-test-suite/12/12.11/12.11-002.js]: add space after assert

File: tests/jerry-test-suite/12/12.11/12.11-004.js [hunk 2]
```diff
@@ -25,4 +25,4 @@ switch ("key") {
     ++counter;
 }
 
-assert(counter == 4 ? 1 : 0);
+assert(counter == 4);
```
Comment (zherczeg) [inline tests/jerry-test-suite/12/12.11/12.11-004.js]: ditto

File: tests/jerry-test-suite/12/12.13/12.13-001.js [hunk 3]
```diff
@@ -19,9 +19,9 @@ function test()
...
-    return 1;
+    return true;
   }
-  return 0;
+  return false;
 }
 
-assert(test());
\ No newline at end of file
+assert(test());
```
Comment (zherczeg) [inline tests/jerry-test-suite/12/12.13/12.13-001.js]: ditto

File: tests/jerry-test-suite/12/12.13/12.13-002.js [hunk 4]
```diff
@@ -19,9 +19,9 @@ function test()
...
-    return 0;
+    return true;
   }
-  return 1;
+  return false;
 }
 
-assert(test);
\ No newline at end of file
+assert(test ());
```
Comment (zherczeg) [inline tests/jerry-test-suite/12/12.13/12.13-002.js]: ditto

File: tests/jerry-test-suite/12/12.13/12.13-003.js [hunk 5]
```diff
@@ -30,9 +30,9 @@ function test()
...
-    return 1;
+    return true;
   }
-  return 0;
+  return false;
 }
 
-assert(test());
\ No newline at end of file
+assert(test());
```
Comment (zherczeg) [inline tests/jerry-test-suite/12/12.13/12.13-003.js]: ok, everywhere

**Comment by zherczeg** [comment]
LGTM, but please fix the spaces

**Comment by ruben-ayrapetyan** [comment]
@zherczeg, I've added spaces before each left parenthesis that follows a letter, in changed files from tests/jerry-test-suite.

**Comment by egavrin** [comment]
Good to me.

**Comment by egavrin** [comment]
Please fix precommit, before merge.

**Comment by ruben-ayrapetyan** [comment]
> Please fix precommit, before merge.

@egavrin, of course. All merges should be performed only if precommit passes successfully.

**Pull request merged**